binding/web/README.md

```diff
@@ -41,7 +39,7 @@ Signup or Login to [Picovoice Console](https://console.picovoice.ai/) to get you
 ### Leopard Models
 
 Leopard requires a model file on initialization. Create a custom model file from [Picovoice Console](https://console.picovoice.ai/cat)
-or you can use the [default model file](/lib/common/leopard_params.pv).
+or you can use the [default model file](https://github.com/Picovoice/leopard/blob/master/lib/common/leopard_params.pv).
```

Comment:
Though this README is good overall. I wonder if it's a good time to bring it more in line with the rest of the repo. Obviously, there will be some differences with the Web binding in particular, but I'd like to see similar titles, sections and flow throughout so that our repo feels more cohesive.